app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :user
```

Comment:
```suggestion
  belongs_to :user`, class_name: 'User'`
```

- [ ] It would be nice to add the class name User to your association setup above, you can check this [article](https://www.microverse.org/blog/how-to-use-associations-and-class-names-in-ruby-on-rails#:~:text=Class_name%20allows%20the%20naming%20of,names%20in%20Ruby%20on%20Rails.) to see the importance of adding class_names in your association